### PR TITLE
Add a quiz question in `python-quiz.ts`

### LINKED ISSUE #15: Adding Python quiz questions
[developerquiz.org](developerquiz.org) currently has over 600 quiz questions. We are looking to expand on the Python questions and we encourage other developers to add their quiz questions to the site.

**Please note: we are only focusing on general Python questions instead of library and framework specific questions.**

You can find the complete list of questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/python-quiz.ts

You can add your own questions to the top of that file.
**Please first check to make sure your question doesn't already exist in the file before creating a PR.**

Here is an example format for the questions. 

```
  {
    Question:
      "In Python, which method can be used to replace parts of a string?",
    Answer: "replace()",
    Distractor1: "switch()",
    Distractor2: "repl()",
    Distractor3: "find()",
    Explanation:
      "If you need to search through a string for a pattern, and replace it with another pattern, you can use the replace() method.",
    Link: "https://www.freecodecamp.org/news/python-string-methods-tutorial-how-to-use-find-and-replace-on-python-strings/",
  },
```
For the `link` field, please make sure to use a freeCodeCamp article, freeCodeCamp YouTube video or official documentation.
If you choose to reference a video, please make sure to include the timestamp for the topic covered.

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#). 

This issue will not be assigned to anyone and will remain open for multiple contributors. 
**Please do not assign yourself to this issue or close it.**

Happy contributing!

### PR DESCRIPTION
**Pull request by MusfiqDehan**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #15

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+   {
+    Question:
+      "In Python, which of the following option will reverse a_string='good' and make it 'doog'?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:4]:
```suggestion
      "In Python, which of the following options will reverse this string: sentence='freeCodeCamp rules'?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "a_string[::-1]",
+    Distractor1: "reversed(a_string)",
+    Distractor2: "a_string.reverse()",
+    Distractor3: "a_string.sort(reverse=True)",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:8]:
```suggestion
    Answer: "sentence[::-1]",
    Distractor1: "reversed(sentence)",
    Distractor2: "sentence.reverse()",
    Distractor3: "sentence.sort(reverse=True)",
```

File: src/data/python-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,15 @@
...
+    Explanation:
+      "<string>[start: stop: step] returns a slice of the string. <string>[:::] returns a copy of the entire string. <string>[::-1] returns a reversed copy of the string.",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]: Let's simplify this a bit.

```suggestion
      "In Python, [::-1] will return a reversed copy of the string.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by MusfiqDehan** [comment]
Hey @jdwilkin4 , I have implemented your suggested changes. Thank you much for guiding me. Feeling happy to contribute to a freecodecamp's project.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by jdwilkin4** [comment]
I removed the `Closes #15 ` from your pull request.
For future PR's please make sure not to close issues that are meant to stay open for multiple contributors. 👍

**Comment by MusfiqDehan** [comment]
Ok, @jdwilkin4 . I didn't understand this before. I will keep this in my mind for the following PRs.

**Pull request merged**